### ISSUE TITLE
PictureBox control does not support UIA accessibility

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3232]
* .NET Core Version: 

.NET Core 5.0
Version: 5.0.100-alpha.1.20073.10
Commit: 29f4d693a9
Runtime Environment:
OS Name: Windows
OS Version: 10.0.18363
OS Platform: Windows
RID: win10-x64
Base Path: C:\Program Files\dotnet\sdk\5.0.100-alpha1-05536
Host (useful for support):
Version: 5.0.0-alpha.1.20072.3
Commit: c3dc1fdfdc

* Have you experienced this same bug with .NET Framework?: 

Yes

**Problem description:**

![image](https://user-images.githubusercontent.com/23213980/81227292-dc5b8900-8ff4-11ea-8696-4fa4f0bb05f9.png)

Accessibility providers:
[pid:23288,providerId:0x70F0E Main:Nested [pid:23476,providerId:0x70F0E Main(parent link):Microsoft: MSAA Proxy (unmanaged:UIAutomationCore.dll)]; Hwnd(parent link):Microsoft: HWND Proxy (unmanaged:uiautomationcore.dll)]"

**Expected behavior:**
There should be UIA accessibility provider from WinForms.

**Minimal repro:**
Create WinForms app with PictureBox control and analyze it using Inspect.

Related item:
This work item is the sub of #2164
